src/top/run/draw_bounding_box.py

```diff
@@ -0,0 +1,69 @@
+"""
+Methods for drawing a bounding box on an image.
+Reference: https://github.com/google-research-datasets/Objectron/blob/master/objectron/dataset/graphics.py
+"""
+
+import cv2
+import numpy as np
+import src.top.run.box_generator as Box
```

Comment:
Why are we importing `box_generator` as `Box` instead of just calling it `box_generator`? We also have a class named `Box` which can be *very* confusing.